crates/snapbox/src/data.rs

```diff
@@ -222,6 +317,8 @@ impl Normalize for NormalizeNewlines {
                 let lines = crate::utils::normalize_lines(&text);
                 Data::text(lines)
             }
+            #[cfg(feature = "json")]
+            DataInner::Json(json) => Data::json(json),
```

Comment:
Should we normalize the newlines in the strings?

Reply:
We probably should normalize the newlines, I hadn't really thought of it that way.

This might require making `normlaize_path` more generic for `Value`. It might need to take in a normalization operation sadly

Reply:
Does it need an operator?  Seems like just functions would be enough and the operator would call that